Assets/Scripts/NpcBehaviors/AngelNpc.cs

```diff
@@ -16,6 +16,7 @@ public class AngelNpc : BaseNpc
     private bool _hasPills;
     private bool _robotGameComplete = false;
     private bool _cowardGameComplete = false;
+    [SerializeField] private GameObject _angelCam;
```

Comment:
For future reference you can make a serialized field like this in the base npc class that all the npcs can refrence if you use the keyword protected or public. It would just be easier than declaring a new vaiable in each npc script. I'm not going to ask you to go back and redo this but I just wanted to let you know in case you didn't!